tests/batchnorm_test.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=no-member, arguments-differ, missing-docstring, invalid-name
+import unittest
+import torch
+from e3nn.batchnorm import BatchNorm
+
+
+class Tests(unittest.TestCase):
+    def test_that_it_runs(self):
+        Rs = [(3, 1), (4, 3)]
+        m = BatchNorm(Rs)
+
+        m.to(dtype=torch.float64)
+        x = torch.randn(10, sum(mul * d for mul, d in Rs), dtype=torch.float64)
+        m(x)
```

Comment:
Should we be testing any properties of the output?

Reply:
Yes we should do more than that :+1: